packages/fe-shared/components/editor.js

```diff
@@ -15,16 +15,36 @@ class Editor extends React.Component {
         super(props);
         this._id = config.id;
         this._entity = config.entity;
+        this._isMobile = config.mobile || false;
```

Comment:
The editors function differently on mobile, so we gotta handle a few things differently.

1. Saving on exit happens in an exit event now, rather than when the entity id changes. Here's the [documentation for reference](https://reactnavigation.org/docs/navigation-events/)
2. Getting the current entity's ID also happens differently. On mobile, it's located in the route params